support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -94,7 +102,7 @@ class CreateSubscriptionController(
       { error =>
         SafeLogger.error(scrub"[${request.uuid}] Failed to create new ${request.body.product.describe}, due to $error")
         error match {
-          case _: RequestValidationError => BadRequest
+          case err: RequestValidationError => BadRequest(err.message)
```

Comment:
I think the "create failed" alarm will still happen for a bad request.  Not 100% sure what the best solution is there though.

Reply:
Yeah I've just investigated a create product failed alarm that was caused by using an email from a blocked domain

![Screenshot 2022-03-10 at 10 54 26](https://user-images.githubusercontent.com/17720442/157647515-58a2bffd-e18e-4a90-ba32-c7ecda29c027.png)

So it's still causing an alarm!

Reply:
What do you think is the best way to prevent the alarm for blocked domains?

Reply:
(I've created a card for this https://trello.com/c/cipTaMhA/333-prevent-alarm-when-user-signs-up-with-a-blocked-email-domain)

Reply:
I've changed this now to go back to doing all the alarming in the action. To avoid alarming for disallowed emails I'm passing the error code for 400s through the `result.header.reasonPhrase`